blocks/subscriptions-block/components/ActionDialog/index.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import PropTypes from '@arc-fusion/prop-types';
+
+import {
+  Button,
+  BUTTON_SIZES,
+  BUTTON_STYLES,
+  BUTTON_TYPES,
+  PrimaryFont,
+} from '@wpmedia/shared-styles';
+
+import './styles.scss';
+
+const ActionDialog = ({
```

Comment:
I thought the plan was to make this component independent of any "Dialog" - How about renaming to be "paywall content"

Reply:
I thought it was the other way... to be able to use this component for dialogs that were not just paywall.

Reply:
Consider these two designs are identical:
https://app.zeplin.io/project/603fa53e2626ed1592e7c0e6/screen/612e29ba7788ff10bab9bf90
https://app.zeplin.io/project/603fa53e2626ed1592e7c0e6/screen/612e2a07071bd3154be8fed6

This could be used for both.